Application/Security.js

```diff
@@ -100,18 +100,117 @@ angular.module('shared')
           }
         );
       },
+      oAuth2Authorize: function(client_id, redirect_uri, response_type, scope, user, callback) {
+        var self = this;
+        
+        switch (response_type) {
```

Comment:
Yeah, eventually this module might support the other oAuth2 authentication modes, which is why I created the switch block. I wanted to make sure that this argument was one of 4 possible strings.